site/src/pages/index.js

```diff
@@ -4,6 +4,7 @@ import SeoComponent from "../components/seo";
 // import Hero from "../sections/Hero";
 import SectionWrapper from "./index.style";
 import PatternsLogo from "../assets/images/brand/SVG/service-mesh-patterns-side_white.svg";
+import Navbar from "../sections/Navbar/index";
```

Comment:
By default the `index` files are imported, you don't need to mention that explicitly.

Reply:
done